packages/lesswrong/components/common/TabNavigationMenu/menuTabs.tsx

```diff
@@ -230,26 +230,31 @@ export const menuTabs: Record<ForumTypeString,Array<MenuTab>> = {
       showOnMobileStandalone: true,
       showOnCompressed: true,
     }, {
-      id: 'sequences',
-      title: 'Sequences',
-      link: '/sequences',
+      id: 'library',
+      title: 'Library',
+      link: '/library',
       iconComponent: LocalLibrary,
-      tooltip: "Collections of posts building on a common theme",
+      tooltip: "Core reading, and sequences of posts building on a common theme",
       showOnMobileStandalone: true,
       showOnCompressed: true,
-    // TODO: Once we get two sequences we want to share, add subitems
-    // }, {
-    //   id: 'fellowship-reading',
-    //   title: 'Fellowship Reading',
-    //   link: '/TODO:',
-    //   tooltip: "TODO: James is cool",
-    //   subItem: true,
-    // }, {
-    //   id: 'motivation',
-    //   title: 'Motivation',
-    //   link: '/TODO:',
-    //   tooltip: "TODO: Aaron's cool",
-    //   subItem: true,
+    }, {
+      id: 'handbook',
+      title: 'The EA Handbook',
+      link: '/handbook',
+      tooltip: "To help you learn the basics of Effective Altruism, we took some of the best writing and made this handbook. Think of it as the textbook you’d get in your first college course. It explains the core ideas of EA, so that you can start applying them to your own life.",
+      subItem: true,
+    }, {
+      id: 'replacing-guilt',
+      title: 'Replacing Guilt',
+      link: '/s/a2LBRPLhvwB83DSGq',
+      tooltip: "Nate Soares writes about replacing guilt with other feelings, exercising self-compassion, and developing confidence — so that we can create a better world.",
+      subItem: true,
+    }, {
+      id: 'most-important-century',
+      title: 'Most Important Century',
+      link: '/posts/TwQzyP3QgttmuTHym/all-possible-views-about-humanity-s-future-are-wild',
```

Comment:
Did you say that we're making these posts into a proper sequence?

Reply:
Yes, though I'm happy to launch as-is, pursuant to Aaron's signoff.